### DISCUSSION TITLE
Adding database session driver for Adonis V5+

### DISCUSSION THREAD

**Discussion by elzdave** [adonisjs/core#4238] [Ideas]
Hello, I'm new to Adonis and really appreciate this great framework 👍🏽

Previously, I use Laravel framework as my main development tools, and using database driver to store user sessions. Now I'm learning Adonis and found out there is no database session driver yet. Is there any plan to create and baked one to the framework core?

Thanks in advance.

**Comment by mastermunj** [comment]
Hi @elzdave, you can follow the guide at https://docs.adonisjs.com/guides/session#extending-from-outside-in

It's easy to setup database driver with this guide.

**Reply by elzdave** [reply]
Hi @mastermunj, yes I already follow the instruction on how to build a custom session driver, and it ran well. But isn't it nice to have it baked into framework core? We can just adjust the config and let the rest handled by Adonis. Seems pretty neat and compact IMHO :)

**Reply by mastermunj** [reply]
Sure. In v6 if it's not default, we can create a community package for the same.

**Reply by darshithedpara** [reply]
Any updates on this? 🤔 
@elzdave  Can you share your implementation via github gist ? - If possible

cc: @mastermunj

**Reply by elzdave** [reply]
> Any updates on this? 🤔 @elzdave Can you share your implementation via github gist ? - If possible
> 
> cc: @mastermunj

Uhm, I haven't code again after I originally post this question. And it seems to be no official session driver for this, so maybe someone interested in bringing the driver into reality? 😃
